starwars/src/components/StarwarCards.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+//child of Starwars
+import { Card, Icon } from 'semantic-ui-react'
+
+
+
+
+const StarwarCards = props => {
+    console.log(props);
+    return(
+    <div className="characterBox">
+        <Card>
+            <h3>Name:</h3>
+   <Card.Content header={props.character.name} />
```

Comment:
Be careful of your white space on the page, it can totally change the readability of your work.